Build/AdditionalProjectTransforms.xslt

```diff
@@ -0,0 +1,5 @@
+<xsl:if test="/Input/Properties/DefineNET4DetectionConstant = 'True'">
+  <PropertyGroup>
+    <DefineConstants Condition=" '$(TargetFrameworkVersion)' == 'v4.0' ">NET4</DefineConstants>
```

Comment:
Mostly for backward compatibility with Windows XP. It's no big deal (~2% of our players are still on XP), but this issue/fix has a limited scope, so I believe it doesn't harm to have that in.